presto-native-execution/etc/config.properties

```diff
@@ -1,5 +1,4 @@
 discovery.uri=http://127.0.0.1:58215
 presto.version=testversion
 http-server.http.port=7777
-shutdown-onset-sec=1
```

Comment:
Let's structure these files so that Prestissimo and Presto-on-Spark directories are peers:

```
presto-native-execution
   presto
      etc
   spark
      etc
```

Reply:
- Should `presto` & `spark` be on-par? shouldn't `presto` the default case & spark be a special case?